src/components/TestToolMenu.js

```diff
@@ -47,6 +47,14 @@ const TestToolMenu = props => (
             />
         </TestToolRow>
 
+        {/* When toggled the app will be forced offline. */}
+        <TestToolRow title="Force offline">
+            <Switch
+                isOn={props.network.shouldForceOffline || false}
+                onToggle={() => Network.setShouldForceOffline(!props.network.shouldForceOffline)}
+            />
+        </TestToolRow>
```

Comment:
Seems fine if we want to start with this - but I personally think this would be more useful if you could quickly switch to offline from anywhere. Maybe with a secret gesture or something. Having to navigate to the `/settings/preferences` screen and back to whatever you were doing before is not ideal.

Reply:
I think the toggle is a good first step for now. I like the gesture idea, as long as its difficult to accidentally trigger.

Reply:
Maybe in an ideal world, we could launch a PopOver containing the debug tools settings page. 

As for how to launch this, 4-5 taps still seems too easy to accidentally trigger IMO. By any chance do we have a list of currently unused recognised gestures?

Reply:
Here's a follow up issue https://github.com/Expensify/Expensify/issues/246352